router/auth.go

```diff
@@ -84,6 +84,42 @@ func authPassThrough(log *log.Logger, w http.ResponseWriter, req *http.Request,
 
 func authInjectECS(log *log.Logger, w http.ResponseWriter, req *http.Request, target *targetPassThroughAuth) bool {
 	req.SetBasicAuth(target.config.Username, target.config.Password)
+	var didWhat string
+	var toWhat string
+	actionUrl := strings.Split(req.URL.String(), "/") // Example: "ecs/ACCESS/FWVERSION/" or "ecs/sam/ForceSim1/"
+	if len(actionUrl) == 5 {
+		switch actionUrl[2] {
+		case "ACCESS":
```

Comment:
I would argue that if we can't parse the URL, then we should not let the call be made. It means someone is trying to send a message to the ECS API that we do not understand (nastiness)